apps/src/applab/ImportScreensDialog.jsx

```diff
@@ -1,5 +1,8 @@
+import React from 'react';
+
 /* eslint-disable react/no-danger */
```

Comment:
this should go at the top of the file, too. Looks like there's a handful of other places where the beginning lines in the file have been reorganized when they shouldn't have been.

Reply:
Ahh sorry. I have the fix for that in another commit. Let me push that.